MovieApp/MovieApp/App/Features/Movies/DataSources/NetworkDataSources/MoviesNetworkDataSource.swift

```diff
@@ -1,12 +1,9 @@
 import Combine
+import Resolver
 
 class MoviesNetworkDataSource: MoviesNetworkDataSourceProtocol {
 
-    private let networkClient: MoviesNetworkClientProtocol
-
-    init(networkClient: MoviesNetworkClientProtocol) {
-        self.networkClient = networkClient
-    }
+    @Injected private var networkClient: MoviesNetworkClientProtocol
```

Comment:
Please do not use @Injected, transfer `networkClient` via the constructor. Change this for all classes.

Reply:
Could you please explain why @injected is bad? Did I use constructors for easier readability inside AppModule or are there some other advantages?